test/unit/objects/MssqlConfig.test.js

```diff
@@ -13,88 +13,91 @@ const sinon = require('sinon');
 chai.use(require('chai-as-promised'));
 chai.use(require('sinon-chai'));
 
-describe('MssqlConfig', () => {
-  describe('.masterDatasource', () => {
-    it("should be 'vin65master'", () => {
+describe('MssqlConfig', async function () {
```

Comment:
just curious if you are running into any issues with `() =>` over `function ()`

Reply:
Since the arrow function doesn't bind the current `this` scope to the function I have run into issues within mocha not having access to the correct scopes. I have reserved using the new arrow syntax within utility functions or any static methods that do not require to be binded to the current instance.